### PR TITLE
SP-292 - Comment widget

### PR DESCRIPTION
**Pull request by fakhergh**

### CODE REVIEW

**Comment by bouthouri** [comment]
Task linked: [SP-292 Comment widget](https://app.clickup.com/t/2427294/SP-292)

File: src/components/CommentBlock/CommentBlock.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { Disqus } from "gatsby-plugin-disqus"
+import React from "react"
+import { useLocation } from "@reach/router"
```
Comment (bouthouri) [inline src/components/CommentBlock/CommentBlock.js:3]: No need for reach router you can get the location from the props

File: src/components/CommentBlock/CommentBlock.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+export function CommentBlock() {
```
Comment (bouthouri) [inline src/components/CommentBlock/CommentBlock.js:5]: pass the location from the page to the comment and in the page get it from the location props

File: src/components/CommentBlock/CommentBlock.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  const location = useLocation()
+
+  const disqusConfig = {
+    url: `${"https://www.sorpetaler.de/" + location.pathname}`,
+    identifier: 1,
+    title: "post1",
```
Comment (bouthouri) [inline src/components/CommentBlock/CommentBlock.js:11]: hardcoded use the page title

File: src/components/CommentBlock/CommentBlock.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import { Disqus } from "gatsby-plugin-disqus"
+import React from "react"
+import { useLocation } from "@reach/router"
+
+export function CommentBlock() {
+  const location = useLocation()
+
+  const disqusConfig = {
+    url: `${"https://www.sorpetaler.de/" + location.pathname}`,
```
Comment (bouthouri) [inline src/components/CommentBlock/CommentBlock.js:9]: hardcoded use the page url

File: gatsby-config.js [hunk 5]
```diff
@@ -27,6 +27,12 @@ module.exports = {
         accessToken: process.env.CONTENTFUL_ACCESS_TOKEN,
       },
     },
+    {
+      resolve: `gatsby-plugin-disqus`,
+      options: {
+        shortname: `sorpetaler`,
```
Comment (bouthouri) [inline gatsby-config.js:33]: hubert not sorpetaler

Comment (fakhergh) [reply gatsby-config.js:33]: @bouthouri hubert shortname is not availale on disqus, this is why I used sorpetaler instead.

Comment (bouthouri) [reply gatsby-config.js:33]: how about sorpetaler/hubert or sorpetaler-hubert

**Review by bouthouri — APPROVED** [review]

**Pull request merged**